extensions/community_extensions/Protected Audience Support.md

```diff
@@ -0,0 +1,125 @@
+# Bid Request Values
+
+## Object: InterestGroupAuctionSupport 
+
+This object allows sellers to signal interest group auction support for an Imp. 
```

Comment:
Would it be possible to indicate where each object is located in each intro paragraph for clarity?

Meaning, writing down that: 
- `InterestGroupAuctionSupport` is located in `BidRequest.imp.ext.igs`
- `InterestGroupAuctionIntent ` is located in `BidResponse.igi`
- `InterestGroupAuctionBuyer ` is located in `BidResponse.igi[i].igb`
- `InterestGroupAuctionSeller ` is located in `BidResponse.igi.igs`